pkg/nimble/contrib/nimble_riot.c

```diff
@@ -33,9 +33,7 @@
 #endif
 
 #ifdef MODULE_NIMBLE_CONTROLLER
-#ifdef CPU_FAM_NRF52
```

Comment:
the ifdef needs to stay, and simply be extended with the `CPU_FAM_NRF51`. Otherwise we get in trouble when building for non Nordic targets (in the future...).